### DISCUSSION TITLE
choose the resolution to meet device needs

### DISCUSSION THREAD

**Discussion by ze-nite** [jellyfin/jellyfin#10115] [General]
Per example a 4k hdr movie hevc, plays well in my pc, and iphone, on the other hand on my brothers phone it does support all the stuff, hdr and hevc but his phone cant really handle the 4k stream and starts lagging, I tried turning down the resolution but it doesnt seem to affect the resolution and only the bitrate.
What I would suggest is on quality selector, being able to choose the original (Direct Stream or play whatever), and then all the resolutions like we have but actually transcoding to the actual resolutions and respective bitrate.

**Comment by njmdietrich** [comment]
If you lower the bitrate enough, the resolution will decrease. I think there is some kind of algorithm that determines the resolution based on bitrate and codec. Also, on jellyfin for android at least you can select the maximum transcoding resolution in the settings.
